scripts/compare-reaction-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { buildSchema } from "graphql"
+import { readFileSync } from "fs"
+const { diff: schemaDiff } = require("@graphql-inspector/core")
+import fetch from "node-fetch"
+
+// If there is a breaking change between the local schema,
+// and the current Reaction one, warn.
+;(async () => {
+  const forcePackageJSON = await (await fetch(
+    "https://raw.githubusercontent.com/artsy/force/release/package.json"
+  )).json()
```

Comment:
The call to `json()` _shouldn't_ return a promise. I think all you need to do is `(await fetch("...")).json()`